docs/src/examples.md

```diff
@@ -1,6 +1,84 @@
 # Examples
 
-## 1D Column examples
+### Advection
+
+The 1D Column advection example in [`examples/column/advect.jl`](https://github.com/CliMA/ClimaCore.jl/blob/main/examples/column/advect.jl).
+
+#### Equations and Discretizations
+
+Follows the continuity equation
+
+```math
+\begin{equation}
+  \frac{\partial \theta}{\partial t} = -\frac{\partial (v \theta)}{\partial z}
+\label{eq:1d-column-advection-continuity}
+\end{equation}
+```
+This is discretized using the following
+
+```math
+\begin{equation}
+  \frac{\partial \theta}{\partial t} \approx - D(V, \theta) 
```

Comment:
Hi @onzerem , here on line 21, you used capital `V`, but below, on line 29 you used lower case `v`. Please be consistent.